modules/sso/sso-pre-launch-actions/configure.sh

```diff
@@ -5,5 +5,8 @@ set -eu
 # shellcheck disable=SC1091
 source "${JBOSS_HOME}/bin/launch/sso-rcfile-definitions.sh"
 
+# CIAM-2404 Ensure using the latest versions of microdnf packages
```

Comment:
Please change the comment to read `Ensure using the latest versions of RPM packages by calling microdnf update`
(the change is not about using the latest version of microdnf packages, but actually about performing RPM packages upgrades to latest available versions)